handlers_test.go

```diff
@@ -176,8 +176,28 @@ func TestHashHandler(t *testing.T) {
 	}
 }
 
+func TestReadiness(t *testing.T) {
+	cfg := defaultCfg
+	cfg.WaitForApp = false
+	e := createEnclave(&cfg)
+	if err := e.Start(); err != nil {
+		t.Fatal(err)
+	}
+	defer e.Stop() //nolint:errcheck
+
+	// Make sure that the Internet-facing Web server is already running because
+	// we didn't ask nitriding to wait for the application.  Give the Web
+	// server 100 milliseconds to start.  This isn't great but there's no
+	// convenient way to check if the Web server is already running.
+	time.Sleep(time.Millisecond * 100)
```

Comment:
You could retry with exponential back-off to speed up the test while verifying the server is responding within some startup interval. I'm not concerned about 100ms though, so this is fine as long as it's reliable.

Reply:
I like that idea! Fixed in 9ac4feb.